_collections/_news/2025-06-24-year-in-review-2025.md

```diff
@@ -0,0 +1,231 @@
+---
+layout: post
+published: true
+title: CSS' Best Year - 2025 End of Year Newsletter
+thumbnail: /assets/images/2025-end-of-year-newsletter/thumbnail.png
+header-image: /assets/images/2025-end-of-year-newsletter/ball-banner.png
+---
+Hello, hello, everyone! Congratulations on surviving this year of university. Whether you're a first year who survived
+the wrath of DSA or a seasoned student who is preparing to graduate, I hope you've all had the chance to catch a well-deserved
+break from exams and studying in general, and have spent plenty of time with your friends and family, wherever in the
+country or even wherever in the world you may be now.
+
+That occasion also marks the time to rewind and reflect on the experiences we've had throughout the entire year - and 
+this is no different for CSS. [Since I've already refreshed everyone on what we've done during semester 1](2024-12-14-2024-25s-semester-1-overview.md), 
+let's go through what we've done throughout semester 2.
+
+# What's been going on
+We held a lot of quirky collaborations with other societies this semester, with societies both well-established and newly
+created. For example, our regular SloLAN game night with GDS, UGG, VR and eSports, a very last-minute entry into the EPS
+Sports Day with the Eagles, CivSoc, MathSoc and many others, but also our `break;` variation with CheeseSoc. We also settled
+the score with Bad Film following the war waged in Semester 1, which we unfortunately lost because somehow they had someone
+who had played Mario Kart Wii competitively for roughly 10 years, so we were set up from the very beginning! Regardless,
```

Comment:
```suggestion
the score with Bad Film following the war waged in Semester 1, which we unfortunately lost because they had someone
who played Mario Kart Wii competitively for roughly 10 years, so we were set up from the very beginning! Regardless,
```